.github/workflows/ci-components.yml

```diff
@@ -0,0 +1,89 @@
+name: CI - Components
+    
+on:
+  push:
+    branches:
+      - main
+  pull_request: {}
+
+env:
+  NODE_VERSION: '16.x'
+
+concurrency:
+   group: ci-components-${{ github.head_ref || github.ref }}
+   cancel-in-progress: true
+
+jobs:
+  test:
+    name: "Tests"
+    runs-on: ubuntu-latest
+
+    steps:
+      - uses: actions/checkout@v2
+      - name: Install Node
+        uses: actions/setup-node@v2
+        with:
+          node-version: ${{ env.NODE_VERSION }}
+          cache: yarn
+          cache-dependency-path: yarn.lock
+      - name: Install Dependencies
+        run: yarn install --frozen-lockfile
+      - name: Lint
+        run: yarn run lint
+        working-directory: packages/components
+      - name: Run Tests
+        run: yarn run test:ember:percy
+        working-directory: packages/components
+        env:
+          PERCY_TOKEN: ${{ secrets.PERCY_TOKEN_COMPONENTS }}
+
+  floating:
+    name: "Floating Dependencies"
+    runs-on: ubuntu-latest
+
+    steps:
+      - uses: actions/checkout@v2
+      - uses: actions/setup-node@v2
+        with:
+          node-version: ${{ env.NODE_VERSION }}
+          cache: yarn
+          cache-dependency-path: yarn.lock
+      - name: Install Dependencies
+        run: yarn install --no-lockfile --non-interactive
+      - name: Run Tests
+        run: yarn run test:ember:percy
+        working-directory: packages/components
+        env:
+          PERCY_TOKEN: ${{ secrets.PERCY_TOKEN_COMPONENTS }}
+
+  try-scenarios:
```

Comment:
design-system (components-ci) 🌔 ~ git co c7459b798af0d30f08aaa1d1ca071ac087d678d6
HEAD is now at c7459b79 Rework /components CI
design-system  🌔 ~ yarn run test:ember-compatibility

can see same error locally:

```
Module not found: Error: Can't resolve '@hashicorp/design-system-tokens/dist/docs/products/tokens.json' in '/private/var/folders/4h/xdzhnlzn31d3ts77p18y5qhm0000gq/T/broccoli-235666awj42qcgRkR/cache-244-webpack_bundler_ember_auto_import_webpack'
```